packages/api/src/command/medical/patient/consolidated-delete.ts

```diff
@@ -69,6 +73,26 @@ async function getResourcesToDelete(
   return resourcesToDelete;
 }
 
+export function filterByDocumentIds(
+  entries: BundleEntry[],
+  documentIds: string[],
+  log = out(`filterByDocumentIds`).log
+): BundleEntry[] {
+  const defaultMsg = `Got ${entries.length} resources from FHIR server`;
+  if (documentIds.length <= 0) {
+    log(`${defaultMsg}, not filtering by documentIds`);
+    return entries;
+  }
+  const isDerivedFromDocRefs = (r: Resource) =>
+    documentIds.some(id => isResourceDerivedFromDocRef(r, id));
+  const filtered =
+    documentIds.length > 0
```

Comment:
Wouldn't this always be true after checking `documentIds.length <= 0`? Is the check necessary?

Reply:
TBH I didn't review this code, I just copy pasted it from [here](https://github.com/metriport/metriport/pull/2756/files#diff-60ab2a379bd8cbc32255b5b03cb5c9848f49a9e9fb01a324f139d3c33214462bL126).

I'd leave as is, otherwise I'd need to test it as well.

Reply:
No worries -- not harming anyone.